RobinKim-SWEngineer/FlutterDevelopment
# PR : Form validation for registration screen

## What is it about?
 - Validation of user inputs for registration
 - Checkpoints
    - Any form is empty
    - Email address is valid ( includes '@')
    - Confirmation password mathes with original one

## How is it written?
 - With `TextEditingController` validating user inputs.

## How to test it?
 - Press the REGISTER button before entering data, it will show that all forms are invalid.
 - As the user enters data, each form will automatically check the validity. 

## Sample image?
![validation_image1](https://user-images.githubusercontent.com/76751931/127298684-4dafa104-6c81-4931-98c7-3514ef2f8456.png)
